File: M8/Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/models/wallet/WalletItems.java
package com.elgin.elginexperience.shipay.models.wallet;

import com.google.gson.annotations.SerializedName;

public class WalletItems {
    @SerializedName("items")
    public WalletItem items;
    @SerializedName("type")
    public String type;
}

File: Exemplos/App_eXperience_Kotlin/app/src/main/java/com/example/e1_elgin_kotlin/shipay/models/wallet/WalletItems.kt
package com.example.e1_elgin_kotlin.shipay.models.wallet

import com.google.gson.annotations.SerializedName

data class WalletItems (
    @SerializedName("items")
    var items: WalletItem,

    @SerializedName("type")
    var type: String
)
